Tasks/KubernetesManifestV0/src/utils/utilities.ts

```diff
@@ -64,21 +64,32 @@ export function annotateChildPods(kubectl: Kubectl, resourceType, resourceName,
     return commandExecutionResults;
 }
 
-export function replaceAllTokens(currentString: string, replaceToken, replaceValue) {
+export function replaceAllTokens(currentString: string, replaceToken: string, replaceValue: string) {
```

Comment:
Rename the function to replaceImageName and give an example of currentString/replaceToken/replacevalue

Reply:
done